<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sample Trimmer Configuration</string>
  </property>
  <layout class="QVBoxLayout" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Effect Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Effect Parameters</string>
     </property>
     <layout class="QFormLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sample Floor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="sampleFloor">
        <property name="suffix">
         <string> dBFS</string>
        </property>
        <property name="minimum">
         <double>-128.000000000000000</double>
        </property>
        <property name="maximum">
         <double>0.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>-70.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="trimSampleStart">
        <property name="text">
         <string>Trim Sample Start</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="trimSampleEnd">
        <property name="text">
         <string>Trim Sample End</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../../synthclone/resources.qrc">
         <normaloff>:/synthclone/images/16x16/close.png</normaloff>:/synthclone/images/16x16/close.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../synthclone/resources.qrc"/>
 </resources>
 <connections/>
</ui>
